crates/bevy_ecs/src/reflect.rs

```diff
@@ -172,6 +172,15 @@ impl ReflectComponent {
     pub fn new(fns: ReflectComponentFns) -> Self {
         Self(fns)
     }
+
+    /// The underlying function pointers implementing methods on `ReflectComponent`.
+    ///
+    /// Prefer using the methods over fetching the function pointers.
+    /// However, this is useful if you want to cache the functions rather than
+    /// repetitively using `TypeRegistry::get()` followed by `data()`.
+    pub fn fn_pointer(&self) -> &ReflectComponentFns {
```

Comment:
Bikeshed: Maybe `raw_fn_pointers` or `raw_functions`? Or at least just `fn_pointers`?

Reply:
I did take a glance at the [std docs for the `fn` type](https://doc.rust-lang.org/stable/std/primitive.fn.html) before opening the PR, and in no places do I see the term "raw". So I kept it to "pointer". I'm going to swap to the plural though.

Reply:
Good call out! Then yeah plural is fine